### PR TITLE
Add privacy measures to allow rejecting for abuse.

### PR DESCRIPTION
**Pull request by mounirlamouri**
This PR adds the new concept, uses it in both requests methods and adds a "Privacy & Security Considerations" section that uses it.

***
<a href="https://pr-preview.s3.amazonaws.com/mounirlamouri/hit-test/pull/73.html" title="Last updated on Jan 17, 2020, 11:49 PM UTC (a8fcfe2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/hit-test/73/a4cae45...mounirlamouri:a8fcfe2.html" title="Last updated on Jan 17, 2020, 11:49 PM UTC (a8fcfe2)">Diff</a>

### CODE REVIEW

**Comment by mounirlamouri** [comment]
@bialpio PTAL

File: index.bs [hunk 1]
```diff
@@ -633,6 +637,21 @@ The information about position and orientation of the intersection point should
 
 Issue: Decide if we need to specify other axes of the coordinate system defined by hit test result's native origin to maintain compatibility between different implementations & differrent AR frameworks.
 
+Privacy & Security Considerations {#privacy-security}
+=================================
+
+<section class="non-normative">
+
+The hit test API can be used to map the user environment with more or less
+precision by sending hit test rays in multitude directions. Hit test has to be
```
Comment (bialpio) [inline index.bs]: s/"multitude"/"multiple", or s/"multitude"/"multitude of" ?

File: index.bs [hunk 2]
```diff
@@ -375,6 +375,8 @@ The {{XRSession}} is extended to contain an associated <dfn for="XRSession">set
 
 The {{XRSession}} is extended to contain an associated <dfn for="XRSession">set of active hit test sources for transient input</dfn> that will be used when computing hit test results for transient input.
 
+The application is considered to make an <dfn>unreasonable amount of requests</dfn> when the amount of currently registered requests or recently made requests is considered too high for a legitimate use of the API. This is an OPTIONAL privacy measure that the user agent MAY take to avoid abuse.
```
Comment (bialpio) [inline index.bs]: Maybe s/"amount of currently registered requests"/"total number of active hit test sources"? We also have a way of being precise on how many hit test sources are currently active (size of the set of active hit test sources + size of the set of active hit test sources).

Also, nit: s/amount/number in the definition and elsewhere where it's used.

Comment (mounirlamouri) [reply index.bs]: Done.

File: index.bs (hunk 1 above)
Comment (mounirlamouri) [reply index.bs]: Done.

**Comment by mounirlamouri** [comment]
Comments applied, PTAL and merge? :)

**Pull request merged**